acos_client/v30/slb/virtual_port.py

```diff
@@ -72,6 +72,9 @@ def _set(self, virtual_server_name, name, protocol, port,
             }, exclude=['template-persist-source-ip', 'template-persist-cookie'])
         }
 
+        sampling_enable = kwargs.get('sampling_enable', [])
```

Comment:
If `sampling_enable` isn't set in an update, it'll be cleared because this defaults to `[]`. How about

```
sampling_enable = kwargs.get('sampling_enable')
if sampling_enable is not None:
    self._set_sampling_enable(sampling_enable, params)
```
or just
```
sampling_enable = kwargs.get('sampling_enable')
if sampling_enable is not None:
    sampling_enable  = map(lambda x: {'counters1': x}, sampling_enable)
...
    "sampling_enable": sampling_enable
```